packages/mongodb-artifact-generator/src/commands/generateJiraPromptResponse.ts

```diff
@@ -0,0 +1,413 @@
+                },
+                created: comment.created,
+                updated: comment.updated,
+              };
+            }),
+          } satisfies FormattedJiraIssue;
+        }
+      );
+
+      logger.appendArtifact(
+        "jiraIssues.formatted.json",
+        JSON.stringify(formattedJiraIssues)
+      );
+
+      // Summarize each issue using a promise pool
+      const summarizeJiraIssue = makeSummarizer({
+        openAi: {
+          client: openAiClient,
+          deployment: OPENAI_CHAT_COMPLETION_DEPLOYMENT,
+        },
+        logger,
+        directions: asBulletPoints(
+          "The task is to summarize the provided Jira issue.",
+          "If there is a notable bug or issue being described, include that in the summary.",
+          "If there are any notable comments, include those in the summary.",
+          "If there are any notable actions that need to be taken to solve, include specific details about those in the summary."
+        ),
+        examples: await Promise.all([
+          loadPromptExamplePairFromFile(
+            path.join(
+              __dirname,
+              "../jiraPromptResponse/examples/WORKPLACE-119.json"
+            )
+          ),
+          loadPromptExamplePairFromFile(
+            path.join(
+              __dirname,
+              "../jiraPromptResponse/examples/ADMIN-10208.json"
+            )
+          ),
+        ]),
+      });
+
+      const summariesByIssueKey = new Map<string, Summary>();
+      const { errors: summarizeJiraIssueErrors } = await PromisePool.for(
+        jiraIssues
+      )
+        .withConcurrency(llmMaxConcurrency)
+        .process(async (issue) => {
+          const summary = await summarizeJiraIssue({
+            input: JSON.stringify(issue),
+          });
+          console.log("summarized issue", issue.key, summary);
+          summariesByIssueKey.set(issue.key, summary);
+        });
+      for (const error of summarizeJiraIssueErrors) {
+        logger.logError(`Error summarizing issue: ${error.item}`);
+        console.log("Error summarizing issue", error.raw);
+      }
+
+      logger.appendArtifact(
+        "summaries.json",
+        JSON.stringify(Object.fromEntries(summariesByIssueKey))
+      );
+
+      // Append summaries to formatted issues
+      const formattedIssuesWithSummaries = formattedJiraIssues.map((issue) => {
+        const summary = summariesByIssueKey.get(issue.key);
+        if (!summary) {
+          throw new Error(`No summary found for issue ${issue.key}`);
+        }
+        return {
+          issue,
+          summary,
+        };
+      }) satisfies FormattedJiraIssueWithSummary[];
+      logger.appendArtifact(
+        "jiraIssues.formattedWithSummaries.json",
+        JSON.stringify(formattedIssuesWithSummaries)
+      );
+
+      // // Generate a list of N questions/prompts for each issue
+      const promptsByIssueKey = new Map<string, string[]>();
+      const generatePrompts = makeGeneratePrompts({
```

Comment:
think should be evaluated but also 🤷

Reply:
https://jira.mongodb.org/browse/EAI-519

Reply:
i think there's some ticket confusion here. looks like you updated the name of an existing ticket plus also created a new one https://jira.mongodb.org/browse/EAI-579 (tho the title on that is wrong i think?)

Reply:
Not sure I see the issue - 519 is for this prompt-response project & 579 is for more generic RAG helpers like "summarize this input" (that are currently defined in/only used in the artifact generator)